### PR TITLE
Ensure kubectl is available in default PATH

### PR DESCRIPTION
**Pull request by dims**
in override-kubectl, we create a /etc/profile.d/kube_env.sh to add
/home/kubernetes/bin/ to PATH. we should source the script to ensure
that kubectl is available

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Comment by neolit123** [comment]
@kubernetes/sig-testing-pr-reviews

**Comment by liggitt** [comment]
cc @seans3

**Comment by liggitt** [comment]
/hold

talked offline… I'd like an ack from @seans3 that this won't cause issues with other consumers of this file. splitting the placement of this between setup scripts and a service definition is really hard to follow

**Comment by liggitt** [comment]
will sync up tomorrow to see if this is ok or if there are reasonable alternatives

**Comment by dims** [comment]
/retest 

(2 flakes)
